pkg/cmd/build/buildkit.go

```diff
@@ -47,7 +50,35 @@ const (
 type buildWriter struct{}
 
 // getSolveOpt returns the buildkit solve options
-func getSolveOpt(buildOptions *types.BuildOptions, okctx OktetoContextInterface) (*client.SolveOpt, error) {
+func getSolveOpt(buildOptions *types.BuildOptions, okctx OktetoContextInterface, secretTempFolder string) (*client.SolveOpt, error) {
```

Comment:
I think this function is also being used within `buildWithDepot` function, why it is within `buildkit.go` file? I think `build` is a better place as it is a common function for 2 different builders.